### PR TITLE
Custom Vendor Re-PR (#45736 Re-PR)

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
🆑
add: custom vendor + price tagger
add: changed the vendomat stock
/🆑

### CODE REVIEW

**Comment by Fikou** [comment]
can you bring back the booze o mat vendor board thanks

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -759,33 +790,200 @@ GLOBAL_LIST_EMPTY(vending_products)
...
+			C = H.get_idcard(TRUE)
+			if(C?.registered_account)
+				private_a = C.registered_account
+				say("\The [src] has been linked to [C].")
+
+	if(compartmentLoadAccessCheck(user))
+		if(istype(I, /obj/item/pen))
+			name = stripped_input(user,"Set name","Name", name, 20)
+			desc = stripped_input(user,"Set description","Description", desc, 60)
+			slogan_list += stripped_input(user,"Set slogan","Slogan","Epic", 60)
```
Comment (cacogen) [inline code/modules/vending/_vending.dm]: can you set the default slogan to "We need a slogan!" for a sick ref to a tarn adams game no one played

**Comment by nemvar** [comment]
What is this? Description for the contents please.

**Comment by PKPenguin321** [comment]
Remake of https://github.com/tgstation/tgstation/pull/45736 which got borked during conflict fixing

**Comment by cacogen** [comment]
put that picture from the old pr in the pr body if you want those thumbs and hearts

**Comment by Ribbl** [comment]
Yes lawd this i needs it

**Pull request merged**